articles/knsh14.re

```diff
@@ -0,0 +1,59 @@
+= 新しい GUI ライブラリ Gio で遊んでみよう！
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)